cmd/kes/main.go

```diff
@@ -43,14 +44,25 @@ func main() {
 	cli.Usage = func() { fmt.Fprint(os.Stderr, usage) }
 
 	var showVersion bool
+	var selfUpdate bool
 	cli.BoolVar(&showVersion, "v", false, "Print version information")
 	cli.BoolVar(&showVersion, "version", false, "Print version information")
+	cli.BoolVar(&selfUpdate, "u", false, "Self updates kes")
+	cli.BoolVar(&selfUpdate, "selfupdate", false, "Self updates kes")
 	cli.Parse(os.Args[1:])
 
 	if showVersion {
 		fmt.Println("kes version", version)
 		return
 	}
+
+	if selfUpdate {
+		if err := update(); err != nil {
+			stdlog.Fatalln(err)
```

Comment:
In case of an error, KES clearly reports that an error occurred:
`stdlog.Fataln("Error: %v", err)`